views/help/forms/form_bug.hbs

```diff
@@ -83,10 +74,9 @@
 	</span>
 	<input class="form-control" type="email" name="replyEmail" placeholder="{{$t "help.placeholder.email"}}" value="{{ userEmail }}" required>
 </label>
-<hr>
 <fieldset>
 	<legend class="control-legend">{{$t "help.global.label.furtherInformation" }}</legend>
-    <label class="consent_label" for="consent">
```

Comment:
when you click on label, it will focus on the referenced field, which is a very nice browser feature, or in this case, it will check / uncheck the checkbox. Was it really necesary to replace it with span?

Reply:
good point, will fix that